scripts/prepare_hierarchical_namespace_test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
I can't seem to get this script working with a storage account level connection string for some reason. I keep getting `Authentication failure. This may be caused by either invalid account key, connection string or sas token value provided for your storage account.`

What does work is if I use `--account-name` and use az login